lib/view/info_view.dart

```diff
@@ -80,6 +81,21 @@ class InfoView extends StatelessWidget {
 
   static Widget _buildSpacer({double height = 16}) => Container(height: height);
 
+  Widget _buildLastUpdatedDate() {
+    if (lastUpdatedTime.isEmpty) {
+      return Container();
+    } else {
+      return Container(
+        margin: const EdgeInsets.all(8.0),
+        child: Text(
+          'Last updated $lastUpdatedTime',
```

Comment:
Probably want the timestamp format to include the timezone? 
The default unix date format is eg `Fri 08 May 2020 14:49:33 AEST` can we just use that?

Reply:
If it takes a lot effort to do time formatting then I think this is okay for now. In the design, it is "9:41am 26 April 2020". We can do as close as possible.